lib/Component.php

```diff
@@ -13,6 +17,8 @@
  * @copyright Copyright (C) fruux GmbH (https://fruux.com/)
  * @author Evert Pot (http://evertpot.com/)
  * @license http://sabre.io/license/ Modified BSD License
+ *
+ * @property VObject\Property\FlatText UID
```

Comment:
I've added this kind of annotations each time a magic property was been used, to improve IDE DX and possibly static analysis.